src/swsssdk/dbconnector.py

```diff
@@ -17,23 +68,28 @@ def __init__(self, **kwargs):
 
 SonicV1Connector.db_map = _connector_map[SonicV1Connector.__name__]['db_map']
 
-if len(SonicV1Connector.db_map) != len({v['db'] for k, v in SonicV1Connector.db_map.items()}):
-    raise RuntimeError("Duplicate DB index detected in configuration.")
-
-
 class SonicV2Connector(DBInterface):
     def __init__(self, **kwargs):
         super(SonicV2Connector, self).__init__(**kwargs)
```

Comment:
>super(SonicV2Connector, self).__init__(**kwargs) [](start = 8, length = 48)

It makes no sense to reuse parent class `__init__`. Suggest add one option `is_unix_socket` for each object with default value true, and remove `isTcpconn` from `connect`

Reply:
done. invoking parent __init__ is necessary , since it initialized some variable/struct